phpstan-baseline.neon

```diff
@@ -730,3 +730,12 @@ parameters:
 			count: 1
 			path: settings/table-settings.php
 
+		-
+			message: "#^Offset 'file' does not exist on array\\{function\\: string, line\\?\\: int, file\\?\\: string, class\\?\\: class\\-string, type\\?\\: '\\-\\>'\\|'\\:\\:', args\\?\\: array, object\\?\\: object\\}\\.$#"
+			count: 1
+			path: include/base.php
+
+		-
+			message: "#^Offset 'line' does not exist on array\\{function\\: string, line\\?\\: int, file\\?\\: string, class\\?\\: class\\-string, type\\?\\: '\\-\\>'\\|'\\:\\:', args\\?\\: array, object\\?\\: object\\}\\.$#"
+			count: 1
+			path: include/base.php
```

Comment:
To prevent this, we can use:
```php
esc_html( $debug[0]['file'] ?? '' ),
absint( $debug[0]['line'] ?? 0 )
```

This is what we've done in `PLL_Model::__call()`.